BUILD.gn

```diff
@@ -235,6 +235,12 @@ group("unittests") {
   }
 }
 
+group("archives") {
```

Comment:
I think this needs to be guarded by `build_engine_artifacts`.

Reply:
Done!